src/utils/Utils.ts

```diff
@@ -1359,7 +1359,7 @@ export default class Utils {
     return crypto.randomBytes(16).toString('hex');
   }
 
-  public static getKeyProperties(): CryptoKeyProperties {
+  public static getDefaultKeyProperties(): CryptoKeyProperties {
     return {
       blockCypher: 'AES',
```

Comment:
You have to be consistent regarding the ciphering settings casing. Store them uppercased or lowercased. You can display them uppercased on the dashboard only and store them lowercased.